examples/distributed/pyg/partition_graph.py

```diff
@@ -3,54 +3,73 @@
 import os.path as osp
 
 import torch
-from ogb.nodeproppred import PygNodePropPredDataset
 
+from benchmark.utils import get_dataset_with_transformation
```

Comment:
It is a bit strange to import from `benchmark` here. Can we change this?

Reply:
Sure, done.